frontend/src/components/Selectors/StateSelector.tsx

```diff
@@ -0,0 +1,30 @@
+import { DropDownMenu } from 'material-ui/DropDownMenu'
```

Comment:
ahhh. I hate how confusing this will be with the one on the add your own dataset page